applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -51,10 +53,19 @@ def __init__(self, Model, settings):
         self.fluid_model_part = self.body_model_part.GetRootModelPart()
         self.trailing_edge_model_part = self.fluid_model_part.CreateSubModelPart("trailing_edge_model_part")
 
+        # model part that contains the elements cut by the wake
+        self.wake_model_part = self.fluid_model_part.CreateSubModelPart("wake_model_part")
+
         # Call the nodal normal calculation util
         KratosMultiphysics.NormalCalculationUtils().CalculateOnSimplex(
             self.fluid_model_part, self.fluid_model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE])
 
+        self.velocity_infinity = [0,0,0]
+        self.velocity_infinity[0] = settings["velocity_infinity"][0].GetDouble()
+        self.velocity_infinity[1] = settings["velocity_infinity"][1].GetDouble()
+        self.velocity_infinity[2] = settings["velocity_infinity"][2].GetDouble()
```

Comment:
Here you are reading the velocity from the parameters, which means that we would have to define it separetely at every process. Actually, the test that you created is reading different v_inf in apply_far_field_process and here (because it takes the default value). This is giving a wrong CL_jump value in the test (it is giving 3.2... instead of 0.32.. because the default v_inf is [1,0,0] and it is not overwritten by the real v_inf which is [10,0,0] in the test).

Now apply_far_field_process.py, compute_lift_process.py and define_wake_process_2d.py are asking for a v_infinity. I think we should move this v_inf elsewere (to be defined in the solver settings maybe?) and write it to ProcessInfo so every process reads the same v_inifinity. 

@philbucher What is the common procedure to do this? Is it fine to define it in the "solver_settings" and then use the ProcessInfo so that every process reads the same v_inf?